### ISSUE TITLE
[CoE Starter Kit - BUG] Production_CoEDashboard usage report shows weekly instead of monthly

### ISSUE THREAD

**Issue by MartinWGT** [microsoft/coe-starter-kit#7571]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

The Power BI report 'Production_CoEDashboard_{date}' has a very useful view of app usage. This shows usage per month but the report is displaying the data by week. As a result data over 2 months can appear in 1 month e.g. Dec and Jan below

![image](https://github.com/microsoft/coe-starter-kit/assets/142497418/82caaccd-1f65-488f-84f6-6ba3f7e78f77)

This can easily be fixed by selecting the Month property in the report as shown below

![image](https://github.com/microsoft/coe-starter-kit/assets/142497418/385bfb09-49ae-4415-85e2-3f06cfb8b6c0)

![image](https://github.com/microsoft/coe-starter-kit/assets/142497418/d0736918-bcac-403c-8a18-4d533fec4194)

### Expected Behavior

The usage report display the monthly data by month instead of by week.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

Production_CoEDashboard_January2024

### What app or flow are you having the issue with?

Production_CoEDashboard_January2024

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

Open the Production_CoEDashboard_January2024 report.
Click on Apps (Deep Dive)
Right click an app and select 'Drill through -> Review app usage'
The app usage report shows weeks instead of months

AB#2174

**Comment by Jenefer-Monroe** [comment]
For the upcoming Feb release, we switched this to be a hierarchy view as shown here.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/b37fca09-e2e5-419d-86ea-504e9b3fccf4)

**Comment by MartinWGT** [comment]
Will we still be able to see monthly totals, as this is a useful especially when forecasting usage for pay as you go set-ups.

**Comment by Jenefer-Monroe** [comment]
PBI hierarchies allow you to drill up to monthly. We can add a feature request around paygo predictions though if you'd like.

**Comment by MartinWGT** [comment]
Yes please. Azure billing gives a forecast for the whole environment, but it would be good to have this at the app level.

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to raise a new issue. We’re happy to help anytime. Thanks again for your engagement!
